api/grpcserver_test.go

```diff
@@ -2768,7 +2800,11 @@ func createServerV2(cfg config.Config, needActPool bool) (*ServerV2, string, err
 	if err != nil {
 		return nil, "", err
 	}
-	svr.core.hasActionIndex = true
+	coreService, ok := svr.core.(*coreService)
+	if !ok {
+		return nil, "", errors.New("fail to cast coreService")
+	}
+	coreService.hasActionIndex = true
```

Comment:
enable `hasActionIndex` by configuring `cfg`

Reply:
move to config.Config?  but hasActionIndex  is set  in  https://github.com/iotexproject/iotex-core/blob/ffb8b7f117cda37ba5e6e281567d5d30a04479e5/api/coreservice.go#L121

Reply:
That's right. As long as you set GatewayPlugin in config, the `hasActionIndex` will be true.